twitter_cldr.gemspec

```diff
@@ -1,28 +1,40 @@
-$:.unshift File.join(File.dirname(__FILE__), 'lib')
-require 'twitter_cldr/version'
+# frozen_string_literal: true
+
+require_relative 'lib/unisec/version'
```

Comment:
```suggestion
require_relative 'lib/twitter_cldr/version'
```

Reply:
I was maybe a bit too quick on the copy paste 😅